src-cpp/rdkafkacpp.h

```diff
@@ -428,8 +428,22 @@ enum ErrorCode {
         ERR_LISTENER_NOT_FOUND = 72,
         /** Topic deletion is disabled */
         ERR_TOPIC_DELETION_DISABLED = 73,
+        /** The leader epoch is older than the broker epoch */
```

Comment:
To be consistent with existing lingua, please remove the "The " prefixes of all these descriptions, and any the's embedded in the text too, so:
"Leader epoch is older than broker epoch"

Reply:
Ok, done.